tutorials/image/cifar10_estimator/cifar10_main.py

```diff
@@ -67,11 +67,23 @@
 
 tf.flags.DEFINE_float('momentum', 0.9, 'Momentum for MomentumOptimizer.')
 
-tf.flags.DEFINE_float('weight_decay', 1e-4, 'Weight decay for convolutions.')
+tf.flags.DEFINE_float('weight_decay', 2e-4, 'Weight decay for convolutions.')
 
 tf.flags.DEFINE_boolean('use_distortion_for_training', True,
                         'If doing image distortion for training.')
 
+tf.flags.DEFINE_boolean('run_experiment', False,
```

Comment:
* Added explanation to Experiments.

* About the "why should we keep estimator option alive" question: I think it's valid to keep both ways to show people that Experiments are not mandatory, and that you can just use the interface if you want to.

**Will Improve README**